ibis/expr/types/generic.py

```diff
@@ -509,6 +510,18 @@ def as_table(self) -> ir.Table:
         table = roots[0].to_expr()
         return table.select(self)
 
+    def to_pandas(self, *args, **kwargs) -> pd.Series:
```

Comment:
For `to_pyarrow`/`to_pyarrow_batches` we moved to keyword-only arguments. Since this is a new API, I think that makes sense to do here too. Only change would be to drop the `*args` from here and the docstring below.

Reply:
Fixed.